synthtool/__main__.py

```diff
@@ -19,9 +19,10 @@
 
 import click
 import pkg_resources
+import synthtool.gcp as gcp
 from synthtool.log import logger
 import synthtool.metadata
-from synthtool import preconfig
+from synthtool import preconfig, copy
```

Comment:
you could add the `gcp` import here too.

```suggestion
from synthtool import copy, gcp, preconfig
```